src/hapi/pipelines/configs/core.yaml

```diff
@@ -38,3 +38,111 @@ org_type:
   resource: "Organization Types (Beta) - CSV"
   format: "csv"
   headers: 2
+
+sector_map:
+  abna: "SHL"
+  abris / nfi: "SHL"
+  abris d'urgence et nfi: "SHL"
+  abris et ana: "SHL"
+  abris: "SHL"
+  abris/ame: "SHL"
+  abris/bna/cccm: "SHL|CCM"
```

Comment:
Oh nooooo - this can be either? How is it presented in 3W? 

Maybe in this case we could have two rows, that differ only in sector?

Reply:
For now I shortened the list to only cover our three countries, and we'll expand as we see these cases!